streams/src/main/java/org/apache/kafka/streams/processor/internals/StreamThread.java

```diff
@@ -1678,14 +1678,14 @@ private void resetOffsets(final Set<TopicPartition> partitions, final Exception
                         addToResetList(
                             partition,
                             seekToBeginning,
-                            "No custom setting defined for topic '{}' using original config 'earliest' for offset reset",
```

Comment:
Have you considered aligning the log message with the `BY_DURATION` (line#1696) case?

Reply:
I just realize that this PR did not do what we want... We want different message for "per topic policy" vs "global policy"... will push an update.